sdmetrics/reports/multi_table/quality_report.py

```diff
@@ -71,68 +37,31 @@ def generate(self, real_data, synthetic_data, metadata, verbose=True):
                 The metadata, which contains each column's data type as well as relationships.
             verbose (bool):
                 Whether or not to print report summary and progress.
+                NOTE: todo
```

Comment:
If this is going to be in another issue can we say what issue here? Otherwise let's just implement it